src/main/java/uk/gov/ons/census/action/model/entity/Case.java

```diff
@@ -77,4 +77,6 @@
   @Column
   @Enumerated(EnumType.STRING)
   private CaseState state;
+
+  @Column private boolean receiptReceived;
```

Comment:
There should be an index on this column so that we don't have to table scan

Reply:
Added